orchagent/aclorch.cpp

```diff
@@ -1228,7 +1229,7 @@ sai_status_t AclOrch::createBindAclTable(AclTable &aclTable, sai_object_id_t &ta
         table_attrs.push_back(attr);
     }
 
-    attr.id =  SAI_ACL_TABLE_ATTR_FIELD_RANGE;
+    attr.id =  SAI_ACL_TABLE_ATTR_FIELD_ACL_RANGE_TYPE;
```

Comment:
SAI does support now range type to be s32list in table creation, so i think the work around can be removed, and pass in proper values